indico/htdocs/sass/partials/_tables.scss

```diff
@@ -43,12 +43,27 @@
     }
 }
 
+@mixin i-table {
```

Comment:
Any reason to use a mixin and not a placeholder class? I think the latter fits better cases like this one.